contracts/V1/DisputeTypes.sol

```diff
@@ -34,12 +34,14 @@ struct Dispute {
     bytes32 genesisStateSnapshotHash;
     /// @notice encoded latest state (latest on-chain state)
     bytes32 latestStateSnapshotHash;
+    /// @dev the blocknumber/height of the latest state
+    uint latestStateHeight;
```

Comment:
Though we're not adding this? For the latestState you have to decode the blocks and prove they are and can't rely on this, since an 'attacker' can put here anything and again force you to decode the blocks to prove it's junk - so you're going to decode them anyway -> this doesn't help :D